src/cswinrt/strings/WinRT.cs

```diff
@@ -98,6 +103,11 @@ private static void InvokeWithObjectParams<T>(this Action<T> func, object arg)
         {
             func.Invoke((T)arg);
         }
+
+        private static void InvokeWithTypedT1<T>(this Action<object, IntPtr> action, T arg1, IntPtr arg2)
+        {
```

Comment:
Also probably want to bring in your draft PR to move all these out of WinRT.cs

Reply:
Yup, will update that too once this is merged! It's also why I left that still as a draft 👍